app/pods/cases/case/subcases/subcase/documents/route.js

```diff
@@ -1,3 +1,45 @@
 import Route from '@ember/routing/route';
+import config from 'fe-redpencil/utils/config';
+import { action } from '@ember/object';
+import { sortPieces } from 'fe-redpencil/utils/documents';
 
-export default Route.extend({});
+export default class DocumentsSubcaseSubcasesRoute extends Route {
```

Comment:
Zou dit niet eerder `subcasesSubcaseDocuments` noemen zodat dit in lijn ligt met de rest van de applicatie?